src/cmd/discard.go

```diff
@@ -12,7 +14,10 @@ var discardCmd = &cobra.Command{
 	Use:   "discard",
 	Short: "Discards the saved state of the previous git-town command",
 	Run: func(cmd *cobra.Command, args []string) {
-		steps.DeletePreviousRunState()
+		err := steps.DeletePreviousRunState()
+		if err != nil {
+			fmt.Printf("cannot delete previous run state: %v", err)
```

Comment:
Why not have a os.Exit here?

Reply:
Good point. Not being able to remove the run state in `git town discard` is error enough to warrant a different exit code. Implemented.